view/frontend/layout/customer_account.xml

```diff
@@ -8,10 +8,11 @@
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
         <referenceBlock name="customer_account_navigation">
-            <block class="Magento\Framework\View\Element\Html\Link\Current" name="customer-account-navigation-privacy-link" after="customer-account-navigation-wish-list-link" ifconfig="gdpr/general/enabled">
+            <block class="Magento\Customer\Block\Account\SortLinkInterface" name="customer-account-navigation-account-privacy-link">
```

Comment:
Please keep `ifconfig="gdpr/general/enabled"` so the link can be enabled from the settings.